src/theme/DocSidebar/styles.module.css

```diff
@@ -1,83 +1,59 @@
 @value large-tablet-breakpoint-query from '~CSS/variables.module.css';
 
-@media large-tablet-breakpoint-query {
-  .sidebar {
-    height: 100vh;
-    overflow-y: auto;
-    position: sticky;
-    top: 0;
-  }
-
-  .sidebar::-webkit-scrollbar {
-    width: 7px;
-  }
-
-  .sidebar::-webkit-scrollbar-track {
-    background: #f1f1f1;
-    border-radius: 10px;
-  }
-
-  .sidebar::-webkit-scrollbar-thumb {
-    background: #888;
-    border-radius: 10px;
-  }
-
-  .sidebar::-webkit-scrollbar-thumb:hover {
-    background: #555;
-  }
-
-  .sidebarLogo {
-    display: flex !important;
-    align-items: center;
-    position: absolute;
-    top: 0;
-    margin: 0 var(--ifm-navbar-padding-horizontal);
-    height: var(--ifm-navbar-height);
-    color: inherit !important;
-    text-decoration: none !important;
-  }
-
-  .sidebarLogo img {
-    margin-right: 0.5rem;
-    height: 2rem;
-  }
-
-  .menu {
-    padding: 0 0.5rem;
-  }
+/* Search */
+:global(.navbar__search) {
+  --search-width: calc(
+    var(--sidebar-width) - 
+    (var(--sidebar-horizontal-padding) * 2 )
+  );
+  border-bottom: 1px solid var(--default-border-color);
 }
 
-.sidebarLogo {
-  display: none;
+:global(.ds-dropdown-menu) {
+  min-width: var(--search-width) !important;
 }
 
-.sidebarMenuIcon {
-  vertical-align: middle;
+:global(.algolia-autocomplete, #search_input_react) {
+  width: 100%;
 }
 
-.sidebarMenuCloseIcon {
-  display: inline-flex;
-  justify-content: center;
-  align-items: center;
-  height: 24px;
-  font-size: 1.5rem;
-  font-weight: var(--ifm-font-weight-bold);
-  line-height: 0.9;
-  width: 24px;
+:global(.algolia-autocomplete) {
+  width: calc( 100% - 10px );
 }
 
-/* Custom */
+:global(#search_input_react) {
+  --icon-size: 16px;
+  background-color: var(--default-background-color);
+  background-repeat: no-repeat;
+  background-size: var(--icon-size) var(--icon-size);
+  background-position: 0 50%;
+  border: 0;
+  color: var(--primary-gray);
+  font-size: 18px;
+  height: 60px !important;
+  padding-left: calc( var(--icon-size) + 10px );
+  -webkit-appearance: none;
+  background-image: url(/img/search.svg);
+}
 
-@media large-tablet-breakpoint-query {
-  .sidebar {
-    height: calc(100vh - 180px);
-  }
+:global(#search_input_react:focus) {
+  outline: none;
 }
 
 .sidebar {
-  --sidebar-gray: #757575;
+  --sidebar-gray: var(--primary-gray);
   font-size: 10px; /* Basis for em */
-  padding-left: 1.5em;
+  outline: none;
```

Comment:
A couple lines in here are not specific to search, but I didn't want to take the time to separate them... It should be pretty clear though which are which